sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -1672,6 +1750,35 @@ public void StartRecognizeInvoicesWithWrongLocale()
 
         #region StartRecognizeCustomForms
 
+        [Test]
+        public async Task StartRecognizeCustomFormsCanAuthenticateWithTokenCredential()
+        {
+            var client = CreateFormRecognizerClient(useTokenCredential: true);
+            await using var trainedModel = await CreateDisposableTrainedModelAsync(useTrainingLabels: true);
```

Comment:
I think it could be worth creating a `useTrainingLabels` test parameter so we can check both recognition with and without labels. We've already found and fixed multiple service and client bugs that happened in one, but not in the other.

(I'm assuming custom form recognition without labels supports AAD as well)

Reply:
Great point. I will add it

Reply:
Test added